extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -236,6 +244,24 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
       this, this.extensions_, this.ampAnalyticsConfig);
+    this.centerCreative_();
```

Comment:
Oh, you're right, of course.  I was thinking of general "futz with iframe" stuff, but yes, for an adsense-specific thing, it belongs in the AdSense impl, and this is the only reasonable place to put it.